### PR TITLE
Display buildings (Ducks v Burger Popup Battle work) for main

### PR DESCRIPTION
**Pull request by 5p0rt5BEArD**
## What 

**Merging Ducks v Burgers Popup Battle work into main**

The work done to support a Ducks v Burger game controlled by building extensions was done in a branch and delivered to 1kx as a deployment from that brancg because some of the core feature changes were rushed through and potentially obselete once we move in a new direction.

Its been decided that these changes will live on a bit longer at least and should become the main code.

There are some known performance issues around support for alwaysActive buildings and potentially some shortcuts taken in code design for their use with a new system for plugins controlling map elements (like tiles other buildings).

These poor-performing, designed-in-a-rush features survive this merge but should become issues to fix up post merge.

Other smaller changes have been kept or discarded as makes sense for each.

### Detail
**Kept:**
- Always Active plugins. (depsite their poor perfromance)
- Display Builing category. (including all potentially temp code from hacked in version for DuckBurger)
- BuildingConstructionBlockNum in schema 
- plugin action buttons are "submit" html (see c41e7855276af02a99f7c3ee21f5417df0ed1eef)
- New maps folder with team-vanilla, arean-1kx and tiny maps
- Removed example buildings that depend on Fixtures folder manifests
- TeamVanilla gitHub action deploy does deploy to azure (rather than just making images)

**Discarded**:
- Loading any maps. Fixtures folder is still the one deployed
- Publishing cli as a release candidate - the 1kx version that used this publish command seems to have become the default, which was not the intention
- Showing author and owner names building panel hade been lost (I think accidently) so reinstated them
- CLI version bump to .17-1kx.0 (its still 0.0.16)
- NewPlayerRule steps that depend on Fixtures folder (these were removed in branch but are back with this merge0

## How
This is a mere of the display-buildings branch into main followed by a series of commits to sanitise it (see details above)

### CODE REVIEW

**Comment by ldunnplaymint** [comment]
Looks like this branch is behind display-buildings by 6 commits. Do we want to update it fully and then get this merged or do it in two stages?

**Review by ldunnplaymint — CHANGES_REQUESTED** [review]
I do not spawn with the ID Card and Acceptance Letter

File: cli/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@playmint/ds-cli",
-  "version": "0.0.16",
+  "version": "0.0.17-1kx.0",
```
Comment (chrisfarms) [inline cli/package.json:3]: think leave this at 16 ... we'll publish 0.17 proper after it's in main

**Review by ldunnplaymint — APPROVED** [review]

**Review by Steedie — APPROVED** [review]
Had a look over the code I worked on and it appears that everything has been added now and is up to date.

**Comment by JackPlaymint** [comment]
There’s a bit of code in the display buildings that are for testing,  but it’s probably useful to keep that in for now.

**Review by JackPlaymint — APPROVED** [review]

**Pull request merged**